nautobot/ipam/forms.py

```diff
@@ -323,12 +323,6 @@ class Meta:
             "date_allocated": DateTimePicker(),
         }
 
-    def __init__(self, *args, **kwargs):
```

Comment:
What is this thing doing over here before lol?

Reply:
```python
    def __init__(self, *args, **kwargs):

        super().__init__(*args, **kwargs)

        self.fields["vrf"].empty_label = "Global"
```